ioctl/newcmd/bc/bcinfo.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package bc
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/iotexproject/iotex-core/ioctl"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+	"github.com/iotexproject/iotex-proto/golang/iotexapi"
+	"github.com/iotexproject/iotex-proto/golang/iotextypes"
+	"github.com/spf13/cobra"
+)
+
+// Multi-language support
+var (
+	bcInfoCmdShorts = map[config.Language]string{
+		config.English: "Get current block chain information",
+		config.Chinese: "获取当前区块链信息",
+	}
+	bcInfoCmdUses = map[config.Language]string{
+		config.English: "info",
+		config.Chinese: "info",
+	}
+)
+
+type infoMessage struct {
+	Node string                `json:"node"`
+	Info *iotextypes.ChainMeta `json:"info"`
+}
+
+func (m *infoMessage) String() string {
+	if output.Format == "" {
+		message := fmt.Sprintf("Blockchain Node: %s\n%s", m.Node, output.JSONString(m.Info))
+		return message
+	}
+	return output.FormatString(output.Result, m)
+}
+
+// NewBCInfoCmd represents the bc info command
+func NewBCInfoCmd(client ioctl.Client) *cobra.Command {
+	bcInfoCmdShort, _ := client.SelectTranslation(bcInfoCmdShorts)
+	bcInfoCmdUse, _ := client.SelectTranslation(bcInfoCmdUses)
+
+	cmd := &cobra.Command{
+		Use:   bcInfoCmdUse,
+		Short: bcInfoCmdShort,
+		Args:  cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			apiServiceClient, err := client.APIServiceClient(ioctl.APIServiceConfig{
+				Endpoint: client.Config().Endpoint,
+				Insecure: (cmd.Flag("insecure") != nil && config.Insecure) || !client.Config().SecureConnect,
```

Comment:
the parameters should be optional. By default, it could look like
`apiServiceClient, err := client.APIServiceClient()` or
`apiServiceClient, err := client.InsecureAPIServiceClient()`

Reply:
fixed